src/scores/probability/crps_impl.py

```diff
@@ -786,19 +786,19 @@ def crps_for_ensemble(
 
 
     .. math::
-        CRPS(x_i, y) = (1 / M) * sum(|x_i - y|) - (1 / 2 * K) * sum(|x_i - x_j|)
+        CRPS(x_i, y) = \\frac{sum(|x_i - y|)}{M} - \\frac{sum(|x_i - x_j|)}{2K}
 
     where the first sum is iterated over 1 <= i <= M and the second sum is iterated over
     1 <= i <= M and 1 <= j <= M.
 
     The value of the constant K in this formula depends on the method:
-        - If `method="ecdf"` then :math:`K = M ** 2`. In this case the CRPS value returned is \
+        - If `method="ecdf"` then :math:`K = M ^ 2`. In this case the CRPS value returned is \
             the exact CRPS value for the emprical cumulation distribution function \
             constructed using the ensemble values.
-        - If `method="fair"` then :math:`K = M * (M - 1)`. In this case the CRPS value returned \
+        - If `method="fair"` then :math:`K = M \\cdot (M - 1)`. In this case the CRPS value returned \
             is the approximated CRPS where the ensemble values can be interpreted as a \
             random sample from the underlying predictive distribution. This interpretation \
-            stems from the formula :math:`\\text{CRPS}(F, Y) = E|X - Y| - E|X - X'|/2`, where X and X' \
+            stems from the formula :math:`\\text{CRPS}(F, Y) = \\mathbb{E} \\cdot |X - Y| - \\frac{1}{2}\\mathbb{E} \\cdot |X - X'|`, where X and X' \
```

Comment:
I think you can remove the `\cdot` delimiter here (surround with brackets), since I'm unsure if it's mathematically correct. Typically its written as `E[X], E(X) or EX`. 

From wiki:
> see: The expected value of a random variable X is often denoted by E(X), E[X], or EX, with E also often stylized as E or 
{\displaystyle \mathbb {E} .}[[1]](https://en.wikipedia.org/wiki/Expected_value#cite_note-1)[[2]](https://en.wikipedia.org/wiki/Expected_value#cite_note-2)[[3]](https://en.wikipedia.org/wiki/Expected_value#cite_note-3)

Reply:
Ah, I misunderstood. I thought it was the expectation value multiplied by that difference, not the expectation value of the difference. I think you're right though, I'll go fix it now, thanks.